frontend/src/utils/action.tsx

```diff
@@ -23,16 +25,29 @@ export const CompareNowAction = ({ videoId }: { videoId: string }) => {
 
 export const AddToRateLaterList = ({ videoId }: { videoId: string }) => {
   const video_id = videoId;
+  const { enqueueSnackbar } = useSnackbar();
   return (
     <Tooltip title="Rate later" placement="left">
       <IconButton
         size="medium"
         color="secondary"
-        onClick={() =>
-          UsersService.usersMeVideoRateLaterCreate({
-            video: { video_id } as Video,
-          })
-        }
+        onClick={async () => {
+          try {
+            const response = await UsersService.usersMeVideoRateLaterCreate({
+              video: { video_id } as Video,
+            });
+            console.log(response);
+            showSuccessAlert(
+              enqueueSnackbar,
+              'The video has been added to your rate later list.'
+            );
+          } catch (error) {
+            showInfoAlert(
+              enqueueSnackbar,
+              'The video is already in your rate later list.'
+            );
+          }
+        }}
```

Comment:
For code readability, I usually like to keep this sort of method, out of the  so that we separate "how something is rendered" and "what the interactions do". 

```JavaScript
export const AddToRateLaterList = ({ videoId }: { videoId: string }) => {
  const video_id = videoId;
  const { enqueueSnackbar } = useSnackbar();

  const handleCreate = async () => {
    [...]
  }

  return (
    <Tooltip title="Rate later" placement="left">
      <IconButton
        size="medium"
        color="secondary"
        onClick={handleCreate}
        style={{ color: '#CDCABC' }}
      >
              <AddIcon />
      </IconButton>
    </Tooltip>
```